### PR TITLE
Add ScanAll for Cosmos DB storage

### PR DESCRIPTION
**Pull request by Torch3333**
This adds the ScanAll operation for the Cosmos DB storage. 
Please have a look.

### CODE REVIEW

File: core/src/main/java/com/scalar/db/storage/cosmos/SelectStatementHandler.java [hunk 1]
```diff
@@ -93,22 +94,15 @@ private List<Record> executeQuery(Operation operation, TableMetadata tableMetada
 
     String query;
     CosmosQueryRequestOptions options;
-    if (ScalarDbUtils.isSecondaryIndexSpecified(scan, tableMetadata)) {
+
+    if (scan instanceof ScanAll) {
+      query = "select * from Record r";
```
[the remark this answers is not in the document]
Comment (Torch3333) [reply core/src/main/java/com/scalar/db/storage/cosmos/SelectStatementHandler.java:99]: It's correct, projections are not sent for a `Scan` and for `Get` requests, they are processed on the client-side on the nonprojected query results. 
I will investigate how to specify them on the query.

[the remark this answers is not in the document]
Comment (brfrn169) [reply core/src/main/java/com/scalar/db/storage/cosmos/SelectStatementHandler.java:99]: Thank you. Anyway we can do that in another PR. I will approve this PR. Thanks.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**